README.md

```diff
@@ -83,6 +88,18 @@ Description is written as the first line of text in your block comment
  */
 ```
 
+#### `fullDescription`
+Full description is written as 2nd first line of text in your block comment
+
+```js
+/*
+ * Normal description
+ *
```

Comment:
What happens if you don't have a blank line between the title and the full description?